### DISCUSSION TITLE
Qucs-S 24.2.0

### DISCUSSION THREAD

**Discussion by ra3xdh** [ra3xdh/qucs_s#665] [Announcements]
# Qucs-S 24.2.0

## New features

* QucsatorRF is now a part of the package; No need to compile and install qucsator manually #370
* Qucs-RFLayout tool could be launched from *Tools* menu #465; this tool must be installed separately: https://github.com/thomaslepoix/Qucs-RFlayout
* Redesign of the Qucsconv GUI #647
* Improved matching tool backported from Qucs-0.0.20 #401
* Added *RF devices* group; backported different RF devices for Qucsator #607

## General imporvements and fixes

* Fixed parameter sweep step issues #186 @ivandi69
* Fixed SPICE devices bounding box #590 #637
* Fixed netlist export from CLI #639
* Fixed parameter sweep simulation with Xyce #624
* Revert explicit WITH_QT6 compile flag because of cmake bug #592
* Fixed tuner issues for German locale #416
* Refactoring in schematic editor #626 #618 @wawuwo

## Localization

* Russian translation update

### Source code

Download the source code file `qucs-s-24.1.0.tar.gz` and build it with `cmake` to try new Qucs-S. 

### Linux

*  DEB and RPM packages for Debian, Ubuntu, Fedora, openSUSE generated with OBS. Find the repository and installation instructions for your distribution here: https://software.opensuse.org/download.html?project=home%3Ara3xdh&package=qucs-s
* The package for Arch is available in AUR: https://aur.archlinux.org/packages/qucs-s
* AppImage for non-mentioned Linux distributions;  use this download link below for file `Qucs-S-x86_64-Linux.AppImage.zip`; unzip and run AppImage;

### Windows installer

Windows installer contains both Qucs-S and Ngspice binaries. Launch the `exe` and follow instructions to install Qucs-S.

* `qucs_s_24_2_0_setup.exe` 64-bit (x86_64) Windows installer including Ngspice

### Windows portable

Windows portable package could be downloaded as zipped distribution. Unpack and launch `qucs-s.exe` from `bin` subdirectory. Ngspice requires a separate installation. 

* `qucs_s_win64_portable_24_2_0.zip` 64-bit (x86_64) portable Windows version;

<hr /><em>This discussion was created from the release <a href='https://github.com/ra3xdh/qucs_s/releases/tag/24.2.0'>Qucs-S 24.2.0</a>.</em>

**Comment by Rmano** [comment]
Thank you! Just a small question: is this `qucsator_rf` a special version of  https://github.com/Qucs/qucsator, or is it the same simulator? I am using qucs-s with a self-compiled `qucsator` for my legacy projects (not RF); if this is the full thing, I consider myself doubly happy...

**Comment by ra3xdh** [comment]
Yes, qucsator_rf is a fork of qucsator. It contains some small improvements for RF simulation: https://github.com/ra3xdh/qucsator_rf/releases/tag/v1.0.0 It has ADMS models disabled because of ADMS deprecation everywhere and difficulties to compile ADMS on some platforms.
